packages/scene-composer/src/layouts/SceneLayout/SceneLayout.tsx

```diff
@@ -84,7 +84,7 @@ const R3FWrapper = (props: { matterportConfig?: MatterportConfig; children?: Rea
       </ContextBridge>
     </MatterportViewer>
   ) : (
-    <UnselectableCanvas shadows dpr={window.devicePixelRatio}>
+    <UnselectableCanvas shadows dpr={window.devicePixelRatio} id='unselectable-canvas'>
```

Comment:
Can we change the id to something like `tm-scene-unselectable-canvas`? This will make sure that it does not conflicts with the one provided by the user when using iot-app-kit or by a Grafana component.

Reply:
Yes, good point I hadn't considered that this could conflict with another component. Changed to your suggestion.